package.json

```diff
@@ -80,21 +80,20 @@
     "semantic-release-monorepo": "^8.0.2",
     "semantic-release-pnpm": "^1.0.2",
     "semver": "^7.3.2",
-    "storybook": "^9.0.15",
```

Comment:
Instead of having storybook as a dev dependency for each package, keep the reference here (and point it to the catalog version). The reason is that there is a bunch of storybook configs at the top level of this monorepo, so storybook can't really be run par package.